pkg/bundle/ruleset/bundle.go

```diff
@@ -73,6 +72,16 @@ func FromBundle(b *bundlev1.Bundle) (*bundlev1.RuleSet, error) {
 			Constraints: []string{},
 		}
 
+		// Process the labels for each secret
+		for label, _ := range p.Labels {
```

Comment:
change it to `for label := range p.Labels {` to be following lint

Reply:
Resolved via https://github.com/elastic/harp/pull/271/commits/5f6d466e892edae5bd080e184bd5b943c158e237